openmdao/core/group.py

```diff
@@ -176,8 +176,12 @@ def connect(self, source, targets, src_indices=None):
             If specified, connect the specified entries of the full
             distributed source value to the target.
         """
-        if isinstance(targets, str):
-            targets = (targets,)
+        src_indices = kwargs.pop('src_indices', None)
+        if len(kwargs) > 0:
+            raise TypeError("connect() got an unexpected keyword argument '{0}'".format(kwargs.popitem()[0]))
```

Comment:
since there could be multiple bad keyword args, should we list them all instead of just the first one?